makerdao/governance-portal-v2
# Chore/sc 1603/hide undelegate button when user has no mkr

### Link to Shortcut ticket:
https://app.shortcut.com/dux-makerdao/story/1603/hide-undelegate-button-when-user-has-no-mkr-delegated

### What does this PR do?
Hides undelegate button if no MKR is delegated

### Steps for testing:
Go to /delegates page

Verify "undelegate" buttons don't show for delegates that user has not delegated to

### Screenshots (if relevant):
<img width="996" alt="Screen Shot 2022-08-30 at 10 24 19 AM" src="https://user-images.githubusercontent.com/5225766/187387944-b3fbe82c-c308-41af-af57-922304225d61.png">
